### PR TITLE
Initial project

### PR DESCRIPTION
**Pull request by NickBalnaves**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## 0.0.1-dev
+
+* Initial release
+
+## 0.0.2-dev
```
Comment (bhl09) [inline CHANGELOG.md:1]: Put the versions in a descending order as (I believe) the standard is the descending order.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## 0.0.1-dev
```
Comment (bhl09) [inline CHANGELOG.md]: Dart follows [sematic versioning](https://semver.org/), where pre-release versions are incremented after the `-dev` tag.
For example: 

1. Initially: `0.0.1-dev.1`
2. Next pre-release will be `0.0.1-dev.2`
3. When all of these are merged to master (stable version), it should be `0.0.1`

**Review by bhl09 — COMMENTED** [review]
Just a small versioning stuff

File: android/src/main/kotlin/com/builttoroam/dual_screen/DualScreenPlugin.kt [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        }
+    }
+
+    override fun onMethodCall(@NonNull call: MethodCall, @NonNull result: Result) {
+        when (call.method) {
+            IS_DUAL_SCREEN_DEVICE -> {
+                result.success(isDualScreenDevice())
+            }
+            IS_APP_SPANNED -> {
+                if (isDualScreenDevice()) {
```
Comment (bhl09) [inline android/src/main/kotlin/com/builttoroam/dual_screen/DualScreenPlugin.kt]: I think we can remove this if statement as `isAppSpanned()` has `isDualScreenDevice()` if statement already?

File: CHANGELOG.md (hunk 2 above)
Comment (NickBalnaves) [reply CHANGELOG.md]: My bad going to have to leave this as i've already published to pub.dev

File: CHANGELOG.md (hunk 1 above)
Comment (NickBalnaves) [reply CHANGELOG.md:1]: Fixed

File: android/src/main/kotlin/com/builttoroam/dual_screen/DualScreenPlugin.kt (hunk 3 above)
Comment (NickBalnaves) [reply android/src/main/kotlin/com/builttoroam/dual_screen/DualScreenPlugin.kt]: Yep, removed

**Pull request merged**